docs/standard-warnings.md

```diff
@@ -0,0 +1,16 @@
+# Standard Warnings
+Some components have scenarios that could cause issues.  Some components require the collector be interacted with in a specific way in order to ensure the component works as intended. This document describes common warnings that may affect a component in the collector. 
+
+Visit a component's README to see if it is affected by any of these standard warnings.
+
+## Unsound Transformations
+Incorrect usage of the component may lead to telemetry data that is unsound i.e. not spec-compliant/meaningless.  This would most likely be caused by converting metric data types or creating new metrics from existing metrics.
+
+## Statefulness
+The component keeps state related to telemetry data and therefore needs all data from a producer to be sent to the same Collector instance to ensure a correct behavior. Examples of scenarios that require state would be computing/exporting delta metrics, tail-based sampling and grouping telemetry.
+
+## Identity Crisis
+The component may change the ['identity' of a metric](https://github.com/open-telemetry/opentelemetry-specification/blob/main//specification/metrics/data-model.md#opentelemetry-protocol-data-model-producer-recommendations).  This could be down either by changing a metrics name, removing attributes, or updating existing attribute values.  Adding attributes to metrics is always safe and does not create an Identity Crisis.
```

Comment:
I thought adding attributes was safe in the sense that adding an attribute could not create a new Identity Conflict with an existing metric.  If that is not true, or there is another scenario that causes the statement to be false, I will remove it.

Reply:
It depends on what you mean by a "metric" and how you define Identity Conflict :-) If we are talking about metric timeseries then I believe the identity of timeseries is defined as [resource attrs, scope attrs, metric name, datapoint attrs]. So, if you add an attribute to the datapoints that's a different timeseries, so the identity is no longer the same. 
cc @bogdandrutu if you have more thoughts.

Reply:
The problem that we are trying to describe AIUI is changing the identity of two different timeseries in a way that makes them have the same identity which is what I believe @TylerHelmuth described as 'identity crisis'. I think that this could be better reflected in the text: the current wording suggests to me that any change in identity is wrong, while it's not necessarily the case. But then I guess Tigran's point makes sense: any change in the identity of a metric could lead to an identity crisis (although, admitedly, operations that drop attributes are more likely to lead to this issue than things like changing metric names or adding attributes)

Reply:
I think identity is defined here https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/data-model.md#timeseries-model

Reply:
I was basing my concept of metric identity on the [OpenTelemetry Protocol data model](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/data-model.md#opentelemetry-protocol-data-model) instead of the timeseries model.  If that is not appropriate let me know.

Should I update the text to be:


## Metric Identity Manipulation
The component may change the ['identity' of a metric](https://github.com/open-telemetry/opentelemetry-specification/blob/main//specification/metrics/data-model.md#opentelemetry-protocol-data-model-producer-recommendations).  This could be done by modifying the metric's name, attributes, or instrumentation scope.  Modifying a metric's identity could result in a metric identity crisis, which caused by two metrics sharing the same name, attributes, and instrumentation scope.

Reply:
that sounds better to me!

Reply:
Both metric identity and timeseries identity may be adjusted by a processor in a way that they result in a conflict. Perhaps mention both.

Also, I have a slight preference for calling this "Identity Conflict" or "Identity Clash". The definition of "crisis" seems a bit less applicable since it is about "time":
>a time of intense difficulty, trouble, or danger

Reply:
+1 to calling this "Identify conflict"